.github/workflows/release.yaml

```diff
@@ -42,7 +35,7 @@ jobs:
           path: |
             ~/go/pkg/mod
             ~/.cache/go-build
-      - uses: crazy-max/ghaction-github-runtime@v1
```

Comment:
How come this is removed?

Reply:
This lets us publish the build cache from buildkit to gha (unless something has changed in the last several months).

Reply:
thanks, reverted